assets/js/pages/Form.tsx

```diff
@@ -69,7 +69,7 @@ function Form() {
         return Promise.reject(response.status);
       })
       .then((data) => {
-        setLinks(JSON.parse(data).webinar_links);
+        setLinks(data.webinar_links);
```

Comment:
Could you make these links clickable and after clicking them redirect you to the proper page?